test/mocha.opts

```diff
@@ -1,6 +1,5 @@
 --bail
 --require should-sinon
---reporter dot
```

Comment:
why did you change the reporter?

Reply:
To have it more readable
Why is it better to have dots ?

Reply:
We changed the reporter a while back because the `dot` reporter has the following advantages:

* the dot reporter filters passing tests out, making errors far more visible
* the default reporter makes it almost impossible to spot warning (such as unhandled promise rejection warnings) or residual console.log outputs amongst the huge wall of text

All in all, it makes the parsing of travis logs much more manageable.

Also, before making this kind of changes, we have the habit of asking if it suits everyone on slack.

Reply:
Okay i see now, thanks for the explaination